### PR TITLE
Add themes 178408971

### PR DESCRIPTION
**Pull request by mikkohei13**
Adds R-package and bumblebee census to themes page https://178408971.dev.laji.fi/theme

### CODE REVIEW

File: projects/laji/src/app/+theme/theme.component.html [hunk 1]
```diff
@@ -48,6 +48,10 @@ <h5>{{ 'navigation.lolife' | translate }}</h5>
       <h5>{{ 'navigation.syke-butterfly' | translate }}</h5>
       <p>{{ 'theme.syke-butterfly.intro' | translate }}</p>
     </a>
+    <a [routerLink]="['/project/' + Global.oldThemeRouting['kimalaiset']] | localize" tabindex="-1" class="navbox">
```
Comment (olzraiti) [inline projects/laji/src/app/+theme/theme.component.html:51]: Could you remove the `tabindex` attribute? They will be removed from all other theme items in https://github.com/luomus/laji/pull/114

**Review by olzraiti — CHANGES_REQUESTED** [review]

File: projects/laji/src/app/+theme/theme.component.html [hunk 2]
```diff
@@ -136,6 +140,10 @@ <h5>{{ 'theme.protax' | translate }}</h5>
       <h5>{{ 'finbif-bib.title' | translate }}</h5>
       <p>{{ 'finbif-bib.intro' | translate }}</p>
     </a>
+    <a [routerLink]="['/about/3123c1'] | localize" tabindex="-1" class="navbox">
```
Comment (olzraiti) [inline projects/laji/src/app/+theme/theme.component.html:143]: Remove `tabindex` here also

**Review by olzraiti — CHANGES_REQUESTED** [review]

**Review by olzraiti — APPROVED** [review]
:+1:

**Review by wkmor1 — CHANGES_REQUESTED** [review]
I think the r-package theme should just link straight to https://luomus.github.io/finbif/

**Review by wkmor1 — APPROVED** [review]

**Pull request merged**